pkg/validation/limits.go

```diff
@@ -418,6 +420,9 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&l.BlockIngestionStatusCode, "limits.block-ingestion-status-code", defaultBlockedIngestionStatusCode, "HTTP status code to return when ingestion is blocked. If 200, the ingestion will be blocked without returning an error to the client. By Default, a custom status code (260) is returned to the client along with an error message.")
 
 	f.IntVar(&l.IngestionPartitionsTenantShardSize, "limits.ingestion-partition-tenant-shard-size", 0, "The number of partitions a tenant's data should be sharded to when using kafka ingestion. Tenants are sharded across partitions using shuffle-sharding. 0 disables shuffle sharding and tenant is sharded across all partitions.")
+
+	_ = l.PatternIngesterTokenizableJSONFields.Set("log,message,msg,msg_,_msg,content")
```

Comment:
Does this mean we need to redefine all of these in the limits if we want to add a new one alongside the existing ones?

Is the default list part of the doc/description so it is easy to reference in that case?

Reply:
Added a default, append and delete lists. So we can better customize the default list per tenant.